src/libraries/System.Text.Json/gen/JsonSourceGenerator.Parser.cs

```diff
@@ -477,6 +477,37 @@ private static bool TryGetClassDeclarationList(INamedTypeSymbol typeSymbol, [Not
                 return true;
             }
 
+            private static string GetClassDeclarationName(INamedTypeSymbol typeSymbol)
+            {
+                if (!typeSymbol.IsGenericType || typeSymbol.TypeArguments.Length == 0)
+                {
+                    return typeSymbol.Name;
+                }
+
+                StringBuilder sb = new StringBuilder();
+
+                sb.Append(typeSymbol.Name);
+                sb.Append('<');
+
+                bool first = true;
+                foreach (ITypeSymbol typeArg in typeSymbol.TypeArguments)
+                {
+                    if (!first)
+                    {
+                        sb.Append(", ");
+                    }
+                    else
+                    {
+                        first = false;
+                    }
```

Comment:
nit: enter after closing brace

Reply:
Done:
https://github.com/dotnet/runtime/pull/71619/files#diff-ddfd61b7ce28a6ed3f103d2d7e5985d323c737e4923200319db566fc57f9c0baR499
https://github.com/dotnet/runtime/pull/71619/files#diff-e03a2e3cb6211e5f996d59a07d1e1f65691278cd97417748ebec9b7e9e7c1cdaR159

Reply:
Sorry, it seems that I misunderstood your point here. I'll just add new line after closing brace in a next commit =)